gateway/modules/eventing/helpers.go

```diff
@@ -179,6 +179,23 @@ func (m *Module) generateProcessedEventRequest(eventID string) *model.UpdateRequ
 	}
 }
 
+func (m *Module) triggerDLQEvent(ctx context.Context, eventDoc *model.EventDocument) error {
+	req := model.QueueEventRequest{
+		Type: fmt.Sprintf("%s%s", utils.DLQEventTriggerSuffix, eventDoc.RuleName),
+		Payload: map[string]interface{}{
+			"id":   eventDoc.ID,
+			"type": eventDoc.Type,
+			"body": eventDoc.Payload,
```

Comment:
```suggestion
			"event_payload": eventDoc.Payload,
```

Reply:
Also, make sure that the `eventDoc.Payload` is an objectand not `[]byte` or `string`.